pandas/tests/indexes/test_base.py

```diff
@@ -29,6 +29,24 @@
 from pandas._libs.lib import Timestamp
 
 
+@pytest.fixture(params=[tm.makeUnicodeIndex(100),
+                        tm.makeStringIndex(100),
+                        tm.makeDateIndex(100),
+                        tm.makePeriodIndex(100),
```

Comment:
you might also want to call the fixture indices to avoid confusion (then in each test that uses you can either rename index -> indices or do a ``index = indices`` at the top)